bpaotu/bpaotu/importer.py

```diff
@@ -247,11 +255,12 @@ def _make_context():
                     attrs[field] = value
                 yield SampleContext(**attrs)
 
-        rows = [t._asdict() for t in soil_contextual_rows(glob(self._import_base + '/base/*.xlsx')[0])]
+        rows = soil_contextual_rows(glob(self._import_base + '/base/*.xlsx')[0])
         mappings = self._load_ontology(DataImporter.soil_ontologies, rows)
         self._session.bulk_save_objects(_make_context())
         self._session.commit()
 
+
     def load_marine_contextual_metadata(self):
         logger.warning("loading Marine Microbes contextual metadata")
 
```

Comment:
revert all of these changes, they don't do anything - we moved this logic into the contextual parser